### PR TITLE
Create test share for storing data used by benchmarks

### PR DESCRIPTION
**Pull request by stvoutsin**
Changes to config & notes on setting up test share, and copying over dataset used by our benchmarks (previously under /user/dcr/)

### CODE REVIEW

**Review by Zarquan — DISMISSED** [review]
Looks good :+1:

File: 20211203-create-test-share.txt [hunk 1]
```diff
@@ -0,0 +1,285 @@
...
+| 6852b819-7395-4786-80c0-06fa9cebcc65 | aglais-user-nch             | 10240 | CEPHFS      | available | True      | cephfsnativetype |      | nova              |
+| fe63568a-d90c-4fb0-8979-07504328809d | aglais-user-stv             |  1024 | CEPHFS      | available | True      | cephfsnativetype |      | nova              |
+| ff351afd-1f06-4d02-9f53-cbe20b0676cc | aglais-user-zrq             |  1024 | CEPHFS      | available | True      | cephfsnativetype |      | nova              |
++--------------------------------------+-----------------------------+-------+-------------+-----------+-----------+------------------+------+-------------------+
+
+
+
+
+# -----------------------------------------------------
+# Create a new 1Tbyte share for Dennis.
```
[the remark this answers is not in the document]
Comment (stvoutsin) [reply 20211203-create-test-share.txt]: Fixed in last commit (https://github.com/wfau/aglais/pull/597/commits/42853a1c0ea6ef38232818ec8e55b479bb31dace)

File: notes/stv/20211203-create-test-share.txt [hunk 2]
```diff
@@ -0,0 +1,288 @@
...
+| 6852b819-7395-4786-80c0-06fa9cebcc65 | aglais-user-nch             | 10240 | CEPHFS      | available | True      | cephfsnativetype |      | nova              |
+| fe63568a-d90c-4fb0-8979-07504328809d | aglais-user-stv             |  1024 | CEPHFS      | available | True      | cephfsnativetype |      | nova              |
+| ff351afd-1f06-4d02-9f53-cbe20b0676cc | aglais-user-zrq             |  1024 | CEPHFS      | available | True      | cephfsnativetype |      | nova              |
++--------------------------------------+-----------------------------+-------+-------------+-----------+-----------+------------------+------+-------------------+
+
+
+
+
+# -----------------------------------------------------
+# Create a new 1Tbyte share for Dennis.
```
[the remark this answers is not in the document]
Comment (stvoutsin) [reply notes/stv/20211203-create-test-share.txt]: Fixed in last commit (https://github.com/wfau/aglais/pull/597/commits/42853a1c0ea6ef38232818ec8e55b479bb31dace)

**Review by Zarquan — APPROVED** [review]
Looks good :+1:

**Pull request merged**